### ISSUE TITLE
Cannot find module 'gulp-accessibility'

### ISSUE THREAD

**Issue by kris-ellery** [yargalot/gulp-accessibility#3]
Whenever I try to run accessibility check I get following error:

```
module.js:338
    throw err;
    ^
Error: Cannot find module 'gulp-accessibility'
```

However, `gulp-accessibility@1.1.2` does exist in `package.json` and is installed in `./node_modules`.

**Comment by yargalot** [comment]
Sorry for the turnaround, should be working now.

just note you'll need to use 

```
access({
      force: true
    })
```

to not fail the build
